CODEOWNERS

```
@@ -1,2 +1,2 @@
 # General code owners
-*      @CPatchane @cedricmessiant
+*      @Crash--
```

```suggestion
*      @Crash-- @edas 
```

Tiens, je pensais déjà y être. J'ai une bonne vision sur certaines parties du code, ajoutons